Project.toml

```diff
@@ -13,13 +14,12 @@ StructArrays = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
 
 [compat]
 Adapt = "3"
+Documenter = "0.27"
```

Comment:
seems like this is not up to date with latest `main` that includes a compat for Documenter v1? could you merge main to this branch?